src/routers/project.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+
+const router = express.Router();
+
+const project = require('../controllers/project')
+
+router.get('/project', project.getAllProjects)
```

Comment:
these endpoints should start with '/' since the root route which is '/projects' is coming from routers index
e.g. => router.get('/', project.getAllProjects), router.get('/:id', project. getProjectByID) and so on